recipes/google-cloud-cpp/2.x/extract_dependencies.py

```diff
@@ -35,13 +37,15 @@
 
 # Used in _generate_proto_requires(): the *.deps files are generated from
 # Bazel and contain a few targets that have incorrect names for CMake.
+#
+# This list maps onto google_cloud_cpp_load_protodeps()'s `targets_substitutions`.
```

Comment:
Thanks a lot for taking the time to add some more context in here! We really appreciate it :)